### DISCUSSION TITLE
Default Active Tabs

### DISCUSSION THREAD

**Discussion by sylcha** [mdbootstrap/TW-Elements#2319] [Support from community]
Hello every one!

I'm using TW-Elements in a project and I have question about [Tabs](https://tw-elements.com/docs/standard/navigation/tabs/).

I have two pages which are shown based on which one of two tabs is clicked and active. My understanding is that the tab that is displayed by default is the one that has `data-te-nav-active` as `li` attribute and `data-te-tabs-active` as `a` attribute.

I would like to set the active page based on a boolean variable `first` : if `first===true`, the first page will be displayed and if `first===false` the second one will be shown.

Does somebody know if there is a way to do that ?

Thanks in advance for your answer.
Sylvain

**Comment by iprzybysz** [comment]
Hi @sylcha, try creating a method which checks your boolean variable and depending on it triggers the `show` method on the target Tab.

**Comment by sylcha** [comment]
I used another approach which seems to work. 

I search for tab elements with `getElementById` and use `setAttribute` to apply `data-te-tab-active` to the appropriate tab based on the boolean variable `first`. I do the same with the `nav` elements...
